README.md

```diff
@@ -67,88 +62,77 @@ To learn more about each framework and their capabilities, check out each framew
 - **[SGLang](components/backends/sglang/README.md)**
 - **[TensorRT-LLM](components/backends/trtllm/README.md)**
 
-Built in Rust for performance and in Python for extensibility, Dynamo is fully open-source and driven by a transparent, OSS (Open Source Software) first development approach.
+# Quick Start
 
-# Installation
+**New to Dynamo?** **[Complete Quickstart Guide](docs/quickstart.md)** (Recommended)
 
-The following examples require a few system level packages.
-Recommended to use Ubuntu 24.04 with a x86_64 CPU. See [docs/support_matrix.md](docs/support_matrix.md)
+## Local Development
 
-## 1. Initial setup
+### Prerequisites
+- Ubuntu 24.04 (recommended) or compatible Linux
+- NVIDIA GPU with CUDA support
+- Docker & Docker Compose
 
-The Dynamo team recommends the `uv` Python package manager, although any way works. Install uv:
-```
+### 1. Install Dynamo
+```bash
+# Install uv (recommended Python package manager)
 curl -LsSf https://astral.sh/uv/install.sh | sh
-```
-
-### Install etcd and NATS (required)
 
-To coordinate across a data center, Dynamo relies on etcd and NATS. To run Dynamo locally, these need to be available.
-
-- [etcd](https://etcd.io/) can be run directly as `./etcd`.
-- [nats](https://nats.io/) needs jetstream enabled: `nats-server -js`.
-
-To quickly setup etcd & NATS, you can also run:
+# Create virtual environment and install Dynamo
+uv venv venv
+source venv/bin/activate
+uv pip install "ai-dynamo[sglang]"  # or [vllm], [trtllm]
 ```
-# At the root of the repository:
-# Edit deploy/docker-compose.yml to comment out "runtime: nvidia" of the dcgm-exporter service if the nvidia container runtime isn't deployed or to be used.
+
+### 2. Start Infrastructure Services
+```bash
+# Start etcd and NATS (required for distributed communication)
 docker compose -f deploy/docker-compose.yml up -d
 ```
 
-## 2. Select an engine
-
-We publish Python wheels specialized for each of our supported engines: vllm, sglang, trtllm, and llama.cpp. The examples that follow use SGLang; continue reading for other engines.
+### 3. Run Your First Model
+```bash
+# Terminal 1: Start frontend
+python -m dynamo.frontend --http-port 8000
 
+# Terminal 2: Start backend worker
+python -m dynamo.sglang --model-path deepseek-ai/DeepSeek-R1-Distill-Llama-8B
```

Comment:
Minor, but [a 0.6B model](https://huggingface.co/Qwen/Qwen3-0.6B) will download much faster. Less waiting around. I believe Qwen3 0.6B is also vllm's default nowadays.